test/new_relic/agent/log_event_aggregator_test.rb

```diff
@@ -15,6 +15,7 @@ def setup
 
       @enabled_config = {
         :'instrumentation.logger' => 'auto',
+        :'instrumentation.logstasher' => 'auto',
```

Comment:
These tests are getting me kind of curious about the Ruby Logger / Logstasher instrumentation interaction. Did you experiment with an application that had both Ruby Logger and Logstasher logs recorded?

Reply:
When both are enabled, we report both loggers outputs. I could see this being unwanted behavior and a suprise to a user. I wonder if loggers outside of Logger should be an opt in. What do you think?

Reply:
The look different enough! I suppose if someone is using both loggers, we can report both. It may be a surprise to some users, but also an easy switch to turn off. Lets keep it enbled to keep with precedence of auto enabling!

Reply:
Sounds good to me!